xExtension-ReadingTime/README.md

```diff
@@ -0,0 +1,19 @@
+Extension pour FreshRSS (https://github.com/FreshRSS)
+
+**v1.0**
+
+Ajoute une estimation du temps de lecture à côté de chaque article.
+Fonctionne sur les affichages de bureau et mobile.
+
+Source: https://framagit.org/Lapineige/FreshRSS_Extension-ReadingTime
```

Comment:
So you are developing it at framagit.org and copy changes to this repository here?

Reply:
Yes. I prefer to keep my files outside of github.
I'll copy the changes (I don't plan to update it that often ^^).

Reply:
I believe git hooks could be used there to automatically keep two repositories in perfect sync.

Reply:
Does it work from gitlab to github ?

Reply:
I have not tried, but I believe so, yes